catalog/ui/catalog-ui-search/src/main/webapp/component/visualization-selector/visualization-selector.less

```diff
@@ -1,31 +1,25 @@
 @{customElementNamespace}visualization-selector {
 
-    > div {
-        padding: @minimumSpacing;
-    }
+  .visualization-choice {
+    white-space: nowrap;
+    padding: @largeSpacing;
+    .grab-cursor;
+  }
+  .visualization-icon {
+      text-align: center;
+      width: @minimumButtonSize;
+  }
 
-    .visualization-choice {
-        display: block;
-        padding: @largeSpacing;
-        height: auto;
-        width: 100%;
-        text-align: left;
-        cursor: move; /* fallback if grab cursor is unsupported */
-        cursor: grab;
-        cursor: -moz-grab;
-        cursor: -webkit-grab;
+  .visualization-icon,
+  .visualization-text {
+    display: inline-block;
```

Comment:
✏️ since those divs are leaf nodes anyway, I think you could leave them as `<span>`s and you wouldn't need to do this.